app/components/attribute_bag.rb

```diff
@@ -30,7 +30,7 @@ def merge(**additional_attrs)
     self.class.new(**new_attrs.deep_merge(additional_attrs))
   end
 
-  def defaults(**default_attrs)
+  def defaults(default_attrs = {})
```

Comment:
@tillprochaska there is more in this file, I'm talking about `slice` and `except`. Apparently it's not executed during tests, can we remove?

Reply:
@roschaefer 

Both are used in components and tested:

https://github.com/tactilenews/100eyes/search?q=slice
https://github.com/tactilenews/100eyes/search?q=except

Reply:
Right! I oversaw they have only one asterisk. So let's merge?